Update qualtrics.qmd (#386)
ubvu/open-handbook: topics/qualtrics.qmd

categories: [Tools]
---

## What is it?

> Qualtrics is a cloud-based subscription and software platform (SaaS) that enables users to create and manage online surveys. The Qualtrics survey tool can support a variety of (complex) survey design requirements by providing such functionalities as different question types, display and branching logic configuration and the use of embedded data and API's.

Reviewer: ```suggestion
Qualtrics is a cloud-based subscription and software platform (SaaS) that enables users to create and manage online surveys. The Qualtrics survey tool can support a variety of (complex) survey design requirements by providing such functionalities as different question types, display and branching logic configuration and the use of embedded data and APIs.
```

## What can it be used for?

All VU students, employees and researchers are eligible to make use of a Qualtrics license, VU's Qualtrics license usage is limited for creating, managing and collecting data for scientific research purposes only.

> Qualtrics should not be used for non-scientific purposes such as the creation of registration or attendance lists, and course evaluation surveys for example; instead users should go to other available tools such as MS Forms.

Reviewer: ```suggestion
Qualtrics should not be used for non-scientific purposes such as the creation of registration or attendance lists and course evaluation surveys for example; instead users should go to other available tools such as [MS Forms](https://www.microsoft.com/en-us/microsoft-365/online-surveys-polls-quizzes).
```

> VU Amsterdam holds an academic license which includes non-standard, advanced features and functions, including but not limited to:

Reviewer: ```suggestion
VU Amsterdam holds an academic licence which includes non-standard, advanced features and functions, including but not limited to:
```

* **FILE UPLOAD** – a non-standard question type that allows respondents to upload a file along with their survey response
* **SIGNATURE QUESTION** – a non-standard question type that presents survey participants with an entry box where they can draw their signature.

> * **API INTEGRATION** – an advanced license feature that can be used to automate repetitive processes inside of Qualtrics or to pass information in and out of Qualtrics.

Reviewer: ```suggestion
* **API INTEGRATION** – an advanced licence feature that can be used to automate repetitive processes inside of Qualtrics or to pass information in and out of Qualtrics.
```

> * **OFFLINE SURVEY APP** – an advanced license feature that comprises a downloadable application available for iOS and Android that allows you to administer surveys on your mobile device or tablet without an internet connection.

Reviewer: ```suggestion
* **OFFLINE SURVEY APP** – an advanced licence feature that comprises a downloadable application available for iOS and Android that allows you to administer surveys on your mobile device or tablet without an internet connection.
```

## How to request access

### Creating and Accessing User Accounts

> VU Amsterdam Qualtrics Central Brand (hereafter VU's Central Brand) at [vuamsterdam.eu.qualtrics.com] allows for user's auto-enrolment, an automated account creation, and registration process. Any user with an enabled VU email address may auto-enroll on VU's Central Brand. The email address will function as the account's username.

Reviewer: ```suggestion
VU Amsterdam Qualtrics Central Brand (hereafter VU's Central Brand) at [vuamsterdam.eu.qualtrics.com](vuamsterdam.eu.qualtrics.com) allows for user's auto-enrolment, an automated account creation and registration process. Any user with an enabled VU email address may auto-enroll on VU's Central Brand. The email address will function as the account's username.
```

> VU's Central Brand makes use of Single-Sign On (SSO). This means that users can log in to Qualtrics using VU's internal login system. Multi-Factor Authentication (MFA) is mandatory to all Qualtrics users utilizing the vuamsterdam brand (i.e. employees and researchers – students will be informed of upcoming project for MFA implementation). MFA acts as an additional layer of security to prevent unauthorised users from accessing accounts and resources, even when the password has been stolen. Qualtrics will use multi-factor authentication to validate user identity and provide quick and convenient access to authorised users.

Reviewer: ```suggestion
VU's Central Brand makes use of Single-Sign On (SSO). This means that users can log in to Qualtrics using VU's internal login system. Multi-Factor Authentication (MFA) is mandatory to all Qualtrics users utilising the vuamsterdam brand (i.e. employees and researchers – students will be informed of upcoming project for MFA implementation). MFA acts as an additional layer of security to prevent unauthorised users from accessing accounts and resources, even when the password has been stolen. Qualtrics will use multi-factor authentication to validate user identity and provide quick and convenient access to authorised users.
```

> ![VU's central brand login page](../public/qualtrics-vu-brand-login-page.png)

Reviewer: ```suggestion
    ![VU's central brand login page](../public/qualtrics-vu-brand-login-page.png)
```

> 4.	Choose “No, I don’t have a preexisting account here” when it is your first time enrolling the central brand

Reviewer: ```suggestion
4. Choose “No, I don’t have a preexisting account here” when it is your first time enrolling the central brand
```

> ![Qualtrics preexisting account](../public/qualtrics/qualtrics-preexisting-account.png)

Reviewer: ```suggestion
    ![Qualtrics preexisting account](../public/qualtrics/qualtrics-preexisting-account.png)
```

> 5.	Click [Sign In]

Reviewer: ```suggestion
5. Click [Sign In]
```

> ![Qualtrics account created successfully](../public/qualtrics/qualtrics-account-created-successfully.png)

Reviewer: ```suggestion
    ![Qualtrics account created successfully](../public/qualtrics/qualtrics-account-created-successfully.png)
```

> 6.	Terms of Service or General Terms and Conditions for Qualtrics Services are presented for review and acceptance. 

Reviewer: ```suggestion
6. Terms of Service or General Terms and Conditions for Qualtrics Services are presented for review and acceptance. 
```

Ask questions to the [Qualtrics community](https://www.qualtrics.com/community/) (platform of all Qualtrics users).

When logging in choose "Sign in with SSO", when asked "Enter your company's Organization ID" fill in `vuamsterdam`.

> ## Available Support

Reviewer: ```suggestion
### Available Support
```

> ## Available Support

Reviewer: ```suggestion
## Contact
```

The following support channels are available:

### For general queries on how to use the tool, including the configuration of advanced features, please refer to the online documentation at qualtrics.com/support 

### For questions concerning survey design and faculty policies with regards to the usage of Qualtrics for your specific research area, please contact your faculty Data Steward/Privacy Champion (see Qualtrics Faculty Contact – Division Managers). Students should contact their course coordinator.    

> ### If you cannot login or encounter an error, for example when using collaboration or email distribution functionalities, please contact the RDM ServiceDesk at rdm@vu.nl (Please do review FAQ)

Reviewer: ```suggestion
If you cannot login or encounter an error, for example when using collaboration or email distribution functionalities, please contact the [RDM Support Desk](mailto:rdm@vu.nl) (an FAQ is not available at the moment, but we are working on it).
```

Ask questions to the [Qualtrics community](https://www.qualtrics.com/community/) (platform of all Qualtrics users).

When logging in choose "Sign in with SSO", when asked "Enter your company's Organization ID" fill in `vuamsterdam`.

## Contact

The following support channels are available:

> For general queries on how to use the tool, including the configuration of advanced features, please refer to the online documentation at the [Qualtrics website](https://www.qualtrics.com/support/).

Reviewer: ```suggestion
* For general queries on how to use the tool, including the configuration of advanced features, please refer to the online documentation at the [Qualtrics website](https://www.qualtrics.com/support/).
```

> 

Reviewer: ```suggestion
```

> For questions concerning survey design and faculty policies with regards to the usage of Qualtrics for your specific research area, please contact your faculty Data Steward/Privacy Champion (see Qualtrics Faculty Contact – Division Managers). Students should contact their course coordinator.    

Reviewer: ```suggestion
* For questions concerning survey design and faculty policies with regards to the usage of Qualtrics for your specific research area, please contact your faculty Data Steward/Privacy Champion (see Qualtrics Faculty Contact – Division Managers). Students should contact their course coordinator.    
```

> 

Reviewer: ```suggestion
```

> If you cannot login or encounter an error, for example when using collaboration or email distribution functionalities, please contact the [RDM Support Desk](mailto:rdm@vu.nl) (an FAQ is not available at the moment, but we are working on it).

Reviewer: ```suggestion
* If you cannot login or encounter an error, for example when using collaboration or email distribution functionalities, please contact the [RDM Support Desk](mailto:rdm@vu.nl) (an FAQ is not available at the moment, but we are working on it).
```
